src/main/java/mate/academy/service/impl/GenericServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import mate.academy.dao.GenericDao;
+import mate.academy.service.GenericService;
+
+public abstract class GenericServiceImpl<T> implements GenericService<T> {
+    @Override
+    public T add(T t) {
+        return getDao().add(t);
+    }
```

Comment:
I think it's not very correct, and ruin Liskov principle, but let's mentor decide